custom_components/tuya_local/devices/petree_cube_litter_box.yaml

```diff
@@ -0,0 +1,151 @@
+name: Petree 2.0 Litter Box
+products:
+  - id: 1kufldu6g6l53rsy
+    manufacturer: Petree
+    model: Petree 2.0
+  - id: witgtqod4iu8tizi
+    manufacturer: Petree
+    model: Petree 2.0
+entities:
+  # Omitted entities:
+  #   - 113 (Pedal angle): Too tedious and is not useful information.
+  #   - 118 ("empty"): Unclear documentation. Also shovel button exists.
+  #   - 124 ("clean"): Unclear documentation. Also shovel button exists.
+  - entity: number
+    name: Delay Clean Time
+    category: config
+    optional: true
+    dps:
+      - id: 5
+        type: integer
+        name: value
+        unit: minute
+        range:
+          min: 1
+          max: 60
+  - entity: switch
+    name: Indicator Light Switch
+    category: config
+    optional: true
+    dps:
+      - id: 102
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Sleep
+    category: config
+    optional: true
+    dps:
+      - id: 103
+        type: boolean
+        name: switch
+  - entity: button
+    name: Deodorization
+    category: config
+    optional: true
+    dps:
+      - id: 104
+        type: boolean
+        name: button
+  - entity: switch
+    name: Do Not Disturb
+    category: config
+    optional: true
+    dps:
+      - id: 105
+        type: boolean
+        name: switch
+  - entity: number
+    name: Do Not Disturb Start Time
+    category: config
+    optional: true
+    dps:
+      - id: 106
+        type: integer
+        name: value
+        unit: minute
+        range:
+          min: 0
+          max: 1439
+  - entity: number
+    name: Do Not Disturb End Time
+    category: config
+    optional: true
+    dps:
+      - id: 107
+        type: integer
+        name: value
+        unit: minute
+        range:
+          min: 0
+          max: 1439
+  - entity: switch
+    name: Beep
+    category: config
+    optional: true
+    dps:
+      - id: 108
+        type: boolean
+        name: switch
+  - entity: sensor
+    name: Notification Status
```

Comment:
These seem like fault conditions rather than general status (except "cat is approaching"), so I would suggest to use a binary_sensor with `class: problem` and no name to take advantage of translations. Standard pattern for such sensors is to have the main sensor dp map to true for fault indications, otherwise false, and a "fault_code" dp pointing to the same dp with no mapping, and optionally a "description" dp with the mapping to readable strings as below.